### PR TITLE
[10.0][ADD] barcodes_generator_location: New module

### PR DESCRIPTION
**Pull request by lasley**
Last, but not least
* Add barcodes generator for stock locations

### CODE REVIEW

File: barcodes_generator_location/models/stock_location.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
```
Comment (tedsalmon) [inline barcodes_generator_location/models/stock_location.py]: `fields` imported but not used.

File: barcodes_generator_location/README.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+   :target: https://runbot.odoo-community.org/runbot/150/10.0
+
+Known issues / Roadmap
+======================
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/pos/issues>`_. In case of trouble, please
```
Comment (tedsalmon) [inline barcodes_generator_location/README.rst]: Wrong repo

File: barcodes_generator_location/README.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```
Comment (tedsalmon) [inline barcodes_generator_location/README.rst]: Old verbiage ;)

**Review by tedsalmon — COMMENTED** [review]
Some minor issues found

**Comment by lasley** [comment]
Thanks @tedsalmon - fixed

**Review by tedsalmon — APPROVED** [review]
👍  Thanks @lasley. LGTM!

File: barcodes_generator_location/README.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+   :alt: License: AGPL-3
+
+=====================================
+Generate Barcodes for Stock Locations
+=====================================
+
+This module expands Odoo functionality, allowing user to generate barcode
+depending on a given barcode rule for Stock Locations.
+
+The barcode will be used as the ``name`` of the lot, which effectively allows
```
Comment (pedrobaeza) [inline barcodes_generator_location/README.rst]: The lot is mentioned here, which doesn't procede here.

File: barcodes_generator_location/README.rst [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+To configure this module, see the 'Configuration' Section of the description
+of the module 'barcodes_generator_abstract'
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to a Picking form:
```
Comment (pedrobaeza) [inline barcodes_generator_location/README.rst]: These instructions don't apply here.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by legalsylvain — APPROVED** [review]
:+1: (after fixing @pedrobaeza remarks).

code review. no test.

thanks !

**Comment by lasley** [comment]
Thanks for the reviews everyone. @pedrobaeza remarks fixed!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Squashing on merging as the extra commits are review fixes

**Pull request merged**